### DISCUSSION TITLE
索尼电视使用kodi通过webdav播放阿里云视频，遥控器连续后退会跳到下一集

### DISCUSSION THREAD

**Discussion by fengchen-github** [AlistGo/alist#6092] [General]
kodi19.5，是alist还是kodi或阿里云的问题？
kodi20.3也这样，kodi20.4版本有问题确认键没反应

**Comment by zhoutao-cn** [comment]
试试这里下载的 https://repo.kodinerds.net/index.php?action=list&scope=cat&item=Binary%20(armeabi-v7a)

**Comment by fengchen-github** [comment]
不知道为什么突然好了，可能是阿里的同学修复了吧
